### PR TITLE
Better exit code

### LINKED ISSUE #62: Feature request: Return another error code when matching patterns
Currently, `phinder find` returns `1` as exit status when matching patterns or occurred an error. If these two cases can distinguish by the status code, it is useful when running in CI.

### PR DESCRIPTION
**Pull request by tomokinakamaru**
- Close #62

---

Exit code list of `phinder find`:

* 0: Success (No error & no violations are found)
* 1: Error (Error & no violations are found)
* 2: Violation (No error & violations are found)
* 3: Error & Violation

### CODE REVIEW

File: src/Cli/Command/FindCommand.php [hunk 1]
```diff
@@ -9,6 +9,14 @@
 
 class FindCommand extends Command
 {
+    private static $_ECODE_SUCCESS = 0;
+
+    private static $_ECODE_ERROR = 1;
+
+    private static $_ECODE_VIOLATION = 2;
+
+    private static $_ECODE_ERROR_VIOLATION = 3;
```
Comment (wata727) [inline src/Cli/Command/FindCommand.php]: What kind of situation is considered as "an error occurred, but violations also are found"?

Comment (wata727) [inline src/Cli/Command/FindCommand.php]: [nitpick] `const` might be better :moyai:

Comment (tomokinakamaru) [reply src/Cli/Command/FindCommand.php]: It is the case where "Phinder fails to parse `foo.php`" and "Phinder find some violations in `bar.php`". Do you think this is overly complex?

Comment (tomokinakamaru) [reply src/Cli/Command/FindCommand.php]: Thank you! I replaced `private static` with `const`

**Pull request merged**